packages/@aws-cdk/aws-events-targets/test/codebuild/codebuild.test.ts

```diff
@@ -16,54 +16,93 @@ test('use codebuild project as an eventrule target', () => {
   rule.addTarget(new targets.CodeBuildProject(project));
 
   // THEN
-  expect(stack).to(haveResource('AWS::Events::Rule', {
```

Comment:
It's usually considered an anti-pattern to modify existing tests (even formatting) as it makes it hard to ensure that this change did not introduce a regression. Do you mind reverting those?

Reply:
I have reverted it. Formatting was the reason. Sorry for that.